doc/ref_arch/openstack/chapters/chapter04.md

```diff
@@ -203,17 +203,35 @@ The [Cinder Configuration]( https://docs.openstack.org/cinder/latest/configurati
 
 [Ceph](https://ceph.io/) is the default CNTT Reference Architecture storage backend and is discussed below.
 
-#### 4.2.4.1. Ceph
+#### 4.2.4.1. Ceph Storage Cluster
+The Ceph storage cluster is deployed on bare metal hardware. The minimal configuration is a cluster of three bare metal servers to ensure High availability. The Ceph Storage cluster consists of 3 main daemon types:
+-	Monitor
+-	OSD (object storage daemon)
+-	RadosGW (Rados Gateway)
 
```

Comment:
+ we need to add Journal

Reply:
@ASawwaf will need to change the sentence preceding the component list to add both Journal and Manager: @samuelhellec do you agree?

Reply:
I agree

Reply:
@samuelhellec Can you make the changes? Thanks

Reply:
done

Reply:
@samuelhellec "The Ceph Storage cluster consists of 4 main daemon types ..." can we change this to something like "The Ceph Storage cluster consists of the following components .." as Journal is not a daemon. Also, I think we should list "Manager."

Reply:
@samuelhellec Manager still needs to be added.

Reply:
@pgoyal01 done